app/resources/v1/pacbio/run_resource.rb

```diff
@@ -59,10 +58,18 @@ class RunResource < JSONAPI::Resource
       # @!attribute [r] sequencing_kit_box_barcodes
       #   @return [Array<String>] the barcodes of the sequencing kits
       attributes :name, :dna_control_complex_box_barcode,
-                 :system_name, :created_at, :state, :comments,
+                 :system_name, :created_at, :state,
                  :pacbio_smrt_link_version_id, :plates_attributes,
                  :adaptive_loading, :sequencing_kit_box_barcodes
 
+      # @!attribute [r] barcodes_and_concentrations
+      #   @return [String] the barcodes and concentrations of the run
+      attribute :barcodes_and_concentrations, readonly: true
+
+      # @!attribute [r] comments [DEPRECATED]
+      #   @return [String] the comments on the run
+      attribute :comments, readonly: true
```

Comment:
Do we still need this comments attribute as we are replacing it with barcodes_and_concentrations?

Reply:
I have left it in as a deprecation just in case there are any side effects.